scripts/progress.py

```diff
@@ -0,0 +1,100 @@
+import os
+from argparse import ArgumentParser
+from functools import lru_cache
+from subprocess import CalledProcessError, check_output
```

Comment:
```suggestion
from subprocess import check_output
```

Remove an unused import